hack/make/deploy/undeploy.mk

```diff
@@ -8,6 +8,8 @@ undeploy/%:
 
 ## Remove the operator with csi-driver, on kubernetes
 undeploy: manifests/crd/helm
+	kubectl delete dynakube --all -n dynatrace
+	kubectl get pods -n dynatrace -o=name | grep dynakube | while read -r POD ; kubectl wait --for=delete $$POD --timeout=60s; done
```

Comment:
we have instructions in our official documentation that should work: https://docs.dynatrace.com/docs/shortlink/guides-k8s-update-uninstall-operator#manifest-uninstall (under "With CSI driver")

if the command there has issues, we need to rethink those instructions as well

Reply:
Thank you, with that command it works perfectly :+1: 
[Fixed here](https://github.com/Dynatrace/dynatrace-operator/pull/3607/commits/d7981fe8fe7c47247b6da587eadf52341036bb7e)